app/javascript/src/ChangePassword/components/PasswordInput.jsx

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import React from 'react'
+import { PasswordField } from 'LoginPage'
+
+type Props = {
+  isRequired: boolean,
+  name: string,
+  fieldId: string,
+  label: string,
+  value: string,
+  isValid: boolean,
+  onChange: () => void,
+  autoFocus?: 'string'
+}
+
+const PasswordInput = ({inputProps}: {inputProps: Props}) => {
```

Comment:
I think this class would be more useful if we moved [this logic](https://github.com/3scale/porta/pull/1664/files#diff-53d51ce55ba27a9ef30bdd59eab90278R20) here instead, the Props being `getInputProps`'s arguments. WDYT?

Reply:
I think it's OK to pass just props here. The logic in `getInputProps` depends on the state of `ChangePassword` component, and I think is better to keep it separated

Reply:
It seems that logic should be inside here instead, because you are hydrating it with your desired props anyways.

Reply:
OK, updated